doc/swish.sty

```diff
@@ -39,7 +39,7 @@
   {\end{argtbl}}
 \newenvironment{tupledef}[1]{\begin{tuple}\codeindex{#1}\end{tuple}\antipar
   \begin{argtbl}}{\end{argtbl}}
-\newcommand{\concern}[1]{\textbf{Concern:} #1}
-\newcommand{\mitigation}{\textbf{Mitigation: }}
```

Comment:
What do you think about \textsc as a middle ground between \textbf and nothing?

Reply:
Small caps was pretty intense. What do you think of slant font? I've pushed it for your consideration.

Reply:
Slant font looks good to me.